tracking/include/pcl/tracking/impl/kld_adaptive_particle_filter_omp.hpp

```diff
@@ -18,65 +20,65 @@ pcl::tracking::KLDAdaptiveParticleFilterOMPTracker<PointInT, StateT>::setNumberO
 }
 
 //////////////////////////////////////////////////////////////////////////////////////////////
-template <typename PointInT, typename StateT> void
-pcl::tracking::KLDAdaptiveParticleFilterOMPTracker<PointInT, StateT>::weight ()
+template <typename PointInT, typename StateT>
+void
+pcl::tracking::KLDAdaptiveParticleFilterOMPTracker<PointInT, StateT>::weight()
 {
   if (!use_normal_)
   {
-#ifdef _OPENMP
-#pragma omp parallel for num_threads(threads_)
-#endif
+#pragma omp parallel for \
+  default(none) \
+  num_threads(threads_)
     for (int i = 0; i < particle_num_; i++)
-      this->computeTransformedPointCloudWithoutNormal (particles_->points[i], *transed_reference_vector_[i]);
-    
-    PointCloudInPtr coherence_input (new PointCloudIn);
-    this->cropInputPointCloud (input_, *coherence_input);
-    if (change_counter_ == 0)
-    {
+      this->computeTransformedPointCloudWithoutNormal(particles_->points[i],
+                                                      *transed_reference_vector_[i]);
+
+    PointCloudInPtr coherence_input(new PointCloudIn);
+    this->cropInputPointCloud(input_, *coherence_input);
+    if (change_counter_ == 0) {
       // test change detector
-      if (!use_change_detector_ || this->testChangeDetection (coherence_input))
-      {
+      if (!use_change_detector_ || this->testChangeDetection(coherence_input)) {
         changed_ = true;
         change_counter_ = change_detector_interval_;
         coherence_->setTargetCloud (coherence_input);
         coherence_->initCompute ();
-#ifdef _OPENMP
-#pragma omp parallel for num_threads(threads_)
-#endif
+#pragma omp parallel for \
```

Comment:
@SunBlack did you skip removal of some `#ifdef _OPENMP` or is this branch based on an older commit?

Reply:
I just merged this with master, after this branch had merge conflicts.